apache-couchdb-mixin/alerts.libsonnet

```diff
@@ -73,14 +73,14 @@
                 (
                   'An average of {{ printf "%%.0f" $value }}ms of request latency has occurred over the last 5 minutes on {{$labels.instance}}, ' +
                   'which is above the threshold of %(alertsWarningRequestLatency5m)sms. '
-                ) % $._config,
+                ) % this.config,
             },
           },
           {
             alert: 'CouchDBHighRequestLatency',
             expr: |||
               sum by(job, instance) (couchdb_request_time_seconds_sum / couchdb_request_time_seconds_count) > %(alertsCriticalRequestLatency5m)s
```

Comment:
I know that you didn't change this but I think there is a bug in the calculation, `sum / count` is in seconds but the config declares the threshold in milliseconds (500), we should multiply the value before the compare operator by `1000`. I think this is the second time I catch this bug, which is not on you but it's there in the original implementations.

```
expr: |||
  sum by(job, instance) (couchdb_request_time_seconds_sum / couchdb_request_time_seconds_count) * 1000 > %(alertsWarningRequestLatency5m)s
||| % this.config,
```

Alternatively, we can change the alert threshold in the config. I am fine with either solution.